tools/depth-quality/depth-quality-model.h

```diff
@@ -143,20 +149,39 @@ namespace rs2
 
             callback_type callback;
 
-            void set_ground_truth(float gt)
+            void set_ground_truth(int gt)
             {
                 std::lock_guard<std::mutex> lock(_m);
-                _ground_truth = gt;
+                _ground_truth_mm = gt;
                 _use_gt = true;
             }
-            void disable_ground_truth() 
+
+            void set_plane_fit(bool found)
             {
                 std::lock_guard<std::mutex> lock(_m);
-                _use_gt = false; 
+                if (_plane_fit != (int)found)
```

Comment:
Not clear why you cast here. maybe use `is_defined(_plane_fit)`

Reply:
I will drop the additional variable completely - all plots will be updated every step